x-pack/test/apm_api_integration/tests/alerts/transaction_error_rate.spec.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApmRuleType } from '@kbn/apm-plugin/common/rules/apm_rule_types';
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import {
+  createApmRule,
+  createIndexConnector,
+  fetchServiceInventoryAlertCounts,
+  fetchServiceTabAlertCount,
+} from './alerting_api_helper';
+import {
+  waitForRuleStatus,
+  waitForDocumentInIndex,
+  waitForAlertInIndex,
+} from './wait_for_rule_status';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const apmApiClient = getService('apmApiClient');
+  const esDeleteAllIndices = getService('esDeleteAllIndices');
+
+  const synthtraceEsClient = getService('synthtraceEsClient');
+
+  registry.when('transaction error rate alert', { config: 'basic', archives: [] }, () => {
+    let ruleId: string;
+    let actionId: string | undefined;
+
+    const INDEX_NAME = 'transaction-error-rate';
```

Comment:
I saw we are using the same variable name in other tests. Can you change it to be more descriptive? If it is the index where the index action documents are stored we can call it `ALERT_ACTION_INDEX_NAME`

```suggestion
    const INDEX_NAME = 'transaction-error-rate';
```